2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_setup.py:_flush():76] Configure stats pid to 82173
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_setup.py:_flush():76] Loading settings from /Users/rajit906/.config/wandb/settings
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_setup.py:_flush():76] Loading settings from /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/settings
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_init.py:_log_setup():520] Logging user logs to /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240625_154025-3/logs/debug.log
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_init.py:_log_setup():521] Logging internal logs to /Users/rajit906/Downloads/Dissertation/Symmetry-invariant-Tractable-Generative-Models/src/wandb/run-20240625_154025-3/logs/debug-internal.log
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_init.py:init():560] calling init triggers
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {'D': 784, 'M': 8000, 'lr': 0.001, 'n_masks': 1, 'num_epochs': 85, 'max_patience': 10, 'batch_size': 64, 'lambda': 0.0}
2024-06-25 15:40:25,753 INFO    MainThread:82173 [wandb_init.py:init():610] starting backend
2024-06-25 15:40:25,754 INFO    MainThread:82173 [wandb_init.py:init():614] setting up manager
2024-06-25 15:40:25,758 INFO    MainThread:82173 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: fork
2024-06-25 15:40:25,759 INFO    MainThread:82173 [wandb_init.py:init():622] backend started and connected
2024-06-25 15:40:25,774 INFO    MainThread:82173 [wandb_run.py:_label_probe_notebook():1328] probe notebook
2024-06-25 15:40:25,774 INFO    MainThread:82173 [wandb_run.py:_label_probe_notebook():1338] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-06-25 15:40:25,774 INFO    MainThread:82173 [wandb_init.py:init():711] updated telemetry
2024-06-25 15:40:25,816 INFO    MainThread:82173 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-06-25 15:41:56,284 ERROR   MainThread:82173 [wandb_init.py:init():770] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-06-25 15:41:56,286 ERROR   MainThread:82173 [wandb_init.py:init():1174] error in wandb.init()
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1164, in init
    return wi.init()
           ^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 776, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-06-25 15:42:09,478 WARNING MsgRouterThr:82173 [router.py:message_loop():77] message_loop has been closed
